Aqua Cam/PhotoCaptureDelegate.swift

```diff
@@ -148,7 +167,6 @@ extension PhotoCaptureProcessor: AVCapturePhotoCaptureDelegate {
 
         PHPhotoLibrary.shared().performChanges({
             let options = PHAssetResourceCreationOptions()
-            options.uniformTypeIdentifier = self.requestedPhotoSettings.processedFileType.map { $0.rawValue }
```

Comment:
/unrelated Let's keep this change regardless this is merged or not.
Apparently unneeded.